amadeus-integration-test/FlightOffersTest.cs

```diff
@@ -39,17 +37,14 @@ public void Post_FlightOffers()
         public void Post_FlightOffersPricing()
         {
             var amadeus = GetAmadeusBuild();
-            var flightOffersPricing = new FlightOffersPricing(amadeus);
-
-            var flightOffersSearch = new FlightOffersSearch(amadeus);
-            FlightOffer[] flightOffersSearches = flightOffersSearch.getFlightOffers(Params.with("originLocationCode", "SYD")
+            FlightOffer[] flightOffersSearches = amadeus.referenceData.shopping.flightOffersSearch.getFlightOffers(Params.with("originLocationCode", "SYD")
             .and("destinationLocationCode", "BKK")
             .and("departureDate", System.DateTime.Now.AddMonths(2).ToString("yyyy-MM-dd"))
             .and("adults", "1")
             .and("max", "2"));
 
             string body = "{\"data\":{\"type\":\"flight-offers-pricing\",\"flightOffers\":[" + flightOffersSearches[0].response.data[0] + "]}}";
-            FlightOfferPricingOutput response = flightOffersPricing.postFlightOffersPricing(new Params(), body);
+            FlightOfferPricingOutput response = amadeus.referenceData.shopping.flightOffers.flightOffersPricing.postFlightOffersPricing(new Params(), body);
```

Comment:
For pricing since the URL is /shopping/flight-offers/pricing, similarly, it shouldn't be under the referenceData.